client/i18n/locales/english/translations.json

```
@@ -305,7 +305,9 @@
     "step-2": "Step 2: Submit your code",
     "submit-public-url": "When you have completed the project, save all the required files into a public repository and submit the URL to it below.",
     "complete-both-steps": "Complete both steps below to finish the challenge.",
-    "runs-in-vm": "The project runs in a virtual machine, complete the user stories described in there and get all the tests to pass to finish step 1."
+    "runs-in-vm": "The project runs in a virtual machine, complete the user stories described in there and get all the tests to pass to finish step 1.",
+    "completed": "Completed",
+    "not-started": "Not started"
```

```suggestion
    "completed": ". Completed",
    "not-started": ". Not started"
```
When I was listening to the button it was a little confusing, because there was no separation between the title and the information about the title.  Adding a full stop made Orca pause briefly and separated the two.

That might help understanding.

---

This is going to be *really* weird on Crowdin.

---

Yeah, I know... I'd like to see if Bruce reckons putting a pause in is worth it and then we can try and come up with something less stupid than a leading full stop.